extensions/wikia/PortableInfoboxBuilder/PortableInfoboxBuilderController.class.php

```diff
@@ -69,6 +69,17 @@ public function getRedirectUrls() {
 		}
 	}
 
+	public function checkIfTemplateExists() {
+		$status = new Status();
+		$response = $this->getResponse();
```

Comment:
I think we can group processing of the request at the beginning and creation of response object along with setting of its vars at the end of this method